fixtures/enum-types/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "enum-types"
+version = "0.22.0"
+authors = ["Firefox Sync Team <sync-team@mozilla.com>"]
+edition = "2021"
+license = "MPL-2.0"
+publish = false
+
+[lib]
+crate-type = ["lib", "cdylib"]
+name = "enum_types"
+
+[dependencies]
+uniffi = {path = "../../uniffi", version = "0.25" }
```

Comment:
FYI, #1859 just landed, so these uniffi references should change from `{ path = ..., version = ... }` to `{ workspace = true }` - check the other fixtures for examples.